sqlglot/parser.py

```diff
@@ -3153,7 +3162,21 @@ def _parse_ordered(self, parse_method: t.Optional[t.Callable] = None) -> exp.Ord
         ):
             nulls_first = True
 
-        return self.expression(exp.Ordered, this=this, desc=desc, nulls_first=nulls_first)
+        if self._match_text_seq("WITH", "FILL"):
+            with_fill = self.expression(exp.WithFill)
```

Comment:
why are you using set instead of just instantiating it?

with_fill = self.expression(..., **{"from": ...

Reply:
Ah, yea can do that too. Will update now thanks.

Reply:
Addressed